core/src/Cabana_AoSoA.hpp

```diff
@@ -352,7 +358,7 @@ class AoSoA
             return;
 
         // Figure out the new capacity.
-        size_type num_soa_alloc = std::floor( n / vector_length );
+        size_type num_soa_alloc = floor( n / vector_length );
```

Comment:
should we just use `floor` everywhere instead of `std::floor` in a `using` statement? I think that should be portable